erpnext/manufacturing/doctype/work_order/work_order.js

```diff
@@ -119,6 +119,10 @@ frappe.ui.form.on("Work Order", {
 			});
 			erpnext.work_order.set_default_warehouse(frm);
 		}
+
+		if (frm.fields_dict["operations"].get_value().length == 0) {
+			frm.fields_dict["operations_section"].collapse();
+		}
```

Comment:
```suggestion
```

This is unnecessary actually. If a section is collapsible then it should be collapsed by default. There's some bug in the framework which is preventing this from happening right now.

Reply:
resolved here: https://github.com/frappe/frappe/pull/15002